scene/animation/animation_player.cpp

```diff
@@ -1325,6 +1327,36 @@ void AnimationPlayer::_animation_changed() {
 	}
 }
 
+void AnimationPlayer::_pause_playing_caches() {
+	for (Set<TrackNodeCache *>::Element *E = playing_caches.front(); E; E = E->next()) {
+		if (E->get()->node && E->get()->audio_playing) {
+			E->get()->node->call("stop");
```

Comment:
Why relying to `call` instead to store the correct `Node` type and just call `stop`? Or just cast as below.

Reply:
This is how it's done currently in `_stop_playing_caches()`:
https://github.com/godotengine/godot/blob/9b6c3a871922d63fe5b92288a373789dc7130b4c/scene/animation/animation_player.cpp#L1328-L1332